Update readme

nekia opened the pull request:
We need to provide a good starting point for users.
Create a new README.md as a landing page for users.

nekia committed:
Add README covered how to get started

Signed-off-by: Atsushi Neki <atsushin@fast.au.fujitsu.com>

nekia committed:
Add steps for e2e testing

Signed-off-by: Atsushi Neki <atsushin@fast.au.fujitsu.com>

anand-jpa commented on Makefile:
```diff
@@ -187,7 +187,7 @@ discover: $(BUILD_DIR)/bin/discover
 token: $(BUILD_DIR)/bin/token
 
-blkvault-repo-docker: $(BUILD_DIR)/image/blkvault-repo/$(DUMMY)
+blkvault-repo-docker: $(BUILD_DIR)/images/blkvault-repo/$(DUMMY)
```
@nekia  I would like to remove all text containing "vault" and replace it with "archiver"

nekia replied:
Rename all 'vault' to 'archiver', including [fabric-block-archiving-testenv repository](https://github.com/nekia/fabric-block-archiving-testenv).

anand-jpa commented on README-E2ETEST.md:
```diff
@@ -0,0 +1,77 @@
+
+## Running end-to-end tests
+
+We using [behave](https://behave.readthedocs.io/en/latest/) python package which is BDD (behaviour-driven development) for e2e testing. uses tests written in a natural language style, backed up by Python code. 
```
Change to "We are using [behave]...."
Also remove the full stop after "for e2e testing".
Instead use "and". That is, "for e2e testing and uses tests written..."

nekia replied:
Fixed it.

nekia committed:
Catch up with the next blockfile to be archived

When stopping archiver node once, this variable which is stored as non-persistent info is cleared.
Changed to retry retrieving blockfile to be archived within finite times.
Metrics of the total amount of archived data is dropped because this information is still available on block archiving repository.

Signed-off-by: Atsushi Neki <atsushin@fast.au.fujitsu.com>

nekia committed:
Work around TODOs

Refactored/Simplified archiveChannelIfNecessary routine.

Signed-off-by: Atsushi Neki <atsushin@fast.au.fujitsu.com>

nekia committed:
Rename blkvault to blkarchiver

Signed-off-by: Atsushi Neki <atsushin@fast.au.fujitsu.com>

nekia committed:
Rename all other 'Vault' to 'Archiver'

Signed-off-by: Atsushi Neki <atsushin@fast.au.fujitsu.com>

nekia committed:
Fix typo in README-E2ETEST.md

Signed-off-by: Atsushi Neki <atsushin@fast.au.fujitsu.com>

anand-jpa reviewed (approved):
Looks good to me.
